Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+                    This will be used in the second part of the mission to get a free intel on (some defenses/factories, TBD)
+
+            Bonus:
+                Build a shield over the crashed ship.
+                All ally engineers repairing the ship survive.
+
+            Unlocked Tech:
+                T2 Static Shield - 3 minutes into the mission
+
+        Mission 2:
+            Intro:
+                Cinematics start at player's ACU, "scanning area aroud for more crystals". Player is assigned to move to the west (that's where the attack came from) to reclaim 3 more crystals that are located there,
+                and to deal with the Aeon bases before they are significant threat.
+
+            Primary:
+                Reclaim 3 more crystals to repair more ship's systems.
+
+                After some minutes, cinematics to the off map at gating Aeon ACU.
+                When player reclaims the crystals or several minutes later, Aeon counter attack coming from the north. This will expand the map to the north/main Aeon base.
+
+            Secondary:
+                Destroy 2 Aeon bases. This is no longer a primary objective.
+
+            Bonus:
+                Destroy over X Submarine Hunters (exact number in the script)
+                Shoot down engineer drop from south base (this will cause one less enemy base in the final part)
+
+            Unlocked Tech:
+                Railgun boat - 5 minutes into the mission
+                T2 Arty - 8 min into the mission
+
+        Mission 3:
+            Intro:
+                Cinematics at the incoming attack from the main base (north-west).
+
+                Frist orbital canon will be on the north island.
+
+            Primary:
+                Defend the Aeon counter attack.
+
+                After few minutes or once player scouts the orbital canon the map will expand to the east (last part).
+
+            Secondary:
+                Defeat the enemy ACU.
+
+            Unlocked Tech:
+                TMD/TML - once player scouts enemy TML (on the north island)
+
+        Mission 4:
+            Primary:
+                Destroy all 4 Orbital canons.
+        
+        Outro:
+            Cinematics at the ship leaving the planet.
+--]]
+
+
+
+-------------
 -- Debriefing
+-------------
+-- Showed on the score screen if player wins
 Debriefing_Win = {
     {text = 'You win', faction = 'UEF'},
 }
 
+-- Showed on the score screen if player loses
 Debriefing_Lose = {
     {text = 'You lose', faction = 'UEF'},
 }
--- Main Story
\ No newline at end of file
+
+
+
+-------------
+-- Win / Lose
+-------------
+-- When the ship dies
+ShipDestroyed = {
+    {text = '[Nichols]: The ship has been killed, mission failed.', vid = 'N03_ShipDestroyed.sfd', bank = 'N03_VO', cue = 'ShipDestroyed', faction = 'UEF'},
+}
+
+-- Player dies
+PlayerDies = {
+    {text = '[Nichols]: Player dead.', vid = 'N03_PlayerDies.sfd', bank = 'N03_VO', cue = 'PlayerDies', faction = 'UEF'},
```

Comment:
Yeah, I think even the coop is not fully ready for just one player dying, as the only thing that happen is ACU explosion that triggers nothing, since the game mode is a sandbox, the army can still play/be given units and stuff.

So first I'd need to find a way of removing the player from the game.